implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -375,17 +391,14 @@ private Stream<String> toMeterLine(Meter meter, BiFunction<Meter, Measurement, S
             .filter(Objects::nonNull);
     }
 
-    private Metric.Builder createMetricBuilder(Meter meter) {
-        return metricBuilderFactory.newMetricBuilder(meter.getId().getName())
-            .setDimensions(fromTags(meter.getId().getTags()))
-            .setTimestamp(Instant.ofEpochMilli(clock.wallTime()))
-            .setUnit(meter.getId().getBaseUnit())
-            .setDescription(meter.getId().getDescription());
-    }
+    private MetricLineBuilder.TypeStep createMetricBuilder(Meter meter) throws MetricException {
+        MetricLineBuilder.TypeStep metricLineBuilder = MetricLineBuilder.create(preConfiguration)
```

Comment:
I'm totally confused now: `MetricLineBuilder.create` results in a `TypeStep` and not a `MetricLineBuilder`? :o

Should not `TypeStep.create` return a `TypeStep`? or `MetricLineBuilder.TypeStep`?

Can we at least do this?
```java
MetricLineBuilder.TypeStep typeStep = MetricLineBuilder.create(preConfiguration)
```
Also the signature:
```java
private MetricLineBuilder.TypeStep createTypeStep(Meter meter) throws MetricException
```

Reply:
Yes, renaming that makes sense. Applied [`98f3514` (#4064)](https://github.com/micrometer-metrics/micrometer/pull/4064/commits/98f35145d1a7ddb07661c85cf00652f1e0dcb456) to fix this.